plugins/arElasticSearchPlugin/lib/arElasticSearchPluginUtil.class.php

```diff
@@ -482,8 +482,8 @@ protected static function getAllObjectStringFields(
                 // Get string fields included in _all
                 elseif (
                     (
-                        !isset($propertyProperties['include_in_all'])
-                        || $propertyProperties['include_in_all']
+                        !isset($propertyProperties['copy_to'])
+                        || ('_all' == $propertyProperties['copy_to'])
```

Comment:
This condition will now copy to `_all` all the mapping fields where you removed `include_in_all: false`. It may be better to add `copy_to: _all` to the fields we really want to include and change this condition to `isset($propertyProperties['copy_to']) && ('_all' == $propertyProperties['copy_to'])`.

Reply:
Updated the mappings.yml file to copy all relevant mappings to _all